sass/DynamicHierarchicalFacet/_DynamicHierarchicalFacetValues.scss

```diff
@@ -87,16 +87,17 @@ button.coveo-dynamic-hierarchical-facet-all {
 
 .coveo-dynamic-hierarchical-facet-value-arrow {
   position: absolute;
+  top: 6px;
```

Comment:
Is the `6px` some arbitrary value ? So that it appears correctly centered or something like this ?

Could we instead arrange for the arrow to align itself using `vertical-align: middle` with the rest of the caption ?

Reply:
I could but top would be -1px for it to be "pixel perfect"